Sources/Base/OAuth2Base.swift

```diff
@@ -462,6 +493,9 @@ open class OAuth2ContextStore {
 	/// Currently used redirect_url.
 	open var redirectURL: String?
 	
+	/// Current code verifier used for PKCE
+	open var codeVerifier: String?
```

Comment:
Better for this to be `public internal(set)`?

Reply:
Done.